src/libraries/Common/src/Interop/Unix/System.Security.Cryptography.Native/Interop.OCSP.cs

```diff
@@ -53,6 +53,22 @@ internal static X509VerifyStatusCode X509ChainGetCachedOcspStatus(SafeX509StoreC
             return response;
         }
 
+        [LibraryImport(Libraries.CryptoNative)]
+        private static partial int CryptoNative_X509ChainHasStapledOcsp(SafeX509StoreCtxHandle storeCtx);
+
+        internal static bool X509ChainHasStapledOcsp(SafeX509StoreCtxHandle storeCtx)
+        {
+            int resp = CryptoNative_X509ChainHasStapledOcsp(storeCtx);
+
+            if (resp == 1)
+            {
+                return true;
+            }
+
+            Debug.Assert(resp == 0, $"Unexpected response from X509ChainHasStapledOcsp: {resp}");
```

Comment:
Looks like the function `CryptoNative_X509ChainHasStapledOcsp` could return -2? I'd also be nervous about assuming hat the `||` operator in C++ result in exactly 0 or exactly 1. Per C++, any non-zero value should be interpreted as _true_.

Reply:
C99 said:

> Semantics
3 The == (equal to) and != (not equal to) operators are analogous to the relational
operators except for their lower precedence. 93) Each of the operators yields 1 if the
specified relation is true and 0 if it is false. 

and

> Semantics
3 The || operator shall yield 1 if either of its operands compare unequal to 0; otherwise, it
yields 0. The result has type int.

So, in C, it seems to be well defined to be 1 or 0.  And, if the compiler disagrees, the assert'll start firing :smile: